contrib/gcppubsub/pkg/dispatcher/dispatcher/reconcile.go

```diff
@@ -41,6 +43,11 @@ import (
 
 const (
 	finalizerName = controllerAgentName
+
+	// Name of the corev1.Events emitted from the reconciliation process
```

Comment:
We should rename these so that they don't match the event names from the Channel reconciler. How about `DispatcherReconciled` or `ChannelDispatcherReconciled`?

Reply:
Done! Used Dispatcher*